src/main/java/billgates/usecases/insert_entry/InsertEntryRequestModel.java

```diff
@@ -12,21 +12,24 @@ public class InsertEntryRequestModel {
     private String from = "";
     private String to = "";
     private String location = "";
+    private final int SplitBillId;
```

Comment:
Convention: variable name should be in the lower camel-case. For example, `SplitBillId` should be changed to `splitBillId`.